jest/private/jest_test.bzl

```diff
@@ -1,18 +1,35 @@
 "Implementation details for jest_test rule"
 
 load("@aspect_rules_js//js:libs.bzl", "js_binary_lib", "js_lib_helpers")
-load("@aspect_bazel_lib//lib:paths.bzl", "relative_file")
+load("@aspect_bazel_lib//lib:paths.bzl", "to_manifest_path", "to_workspace_path")
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
 
 _attrs = dicts.add(js_binary_lib.attrs, {
+    "auto_configure_reporters": attr.bool(
+        doc = """Let rules_jest configure reporters for Bazel test and xml test logs.
+The `default` reporter is used for the standard test log and jest-junit is used
+for the xml log.
+
+Set this to False to use the reporters configured in the jest config file.
+
+jest_test always sets the `JEST_JUNIT_OUTPUT_FILE` env var to point to the Bazel
```

Comment:
```
jest_test always sets the `JEST_JUNIT_OUTPUT_FILE` env var to point to the where Bazel
expects jest to write the xml test log, so if you set `auto_configure_reporters` to False and
configure jest-junit in the jest config file, it will output the junit xml file where Bazel expects by default.""",
```